tripfile.js

```diff
@@ -39,7 +39,8 @@ export async function build() {
 	await src.read()
 		.then(preprocess)
 		.then(optimise)
-		.then(dist.write);
+		.then(dist.write)
+		.catch(console.error);
```

Comment:
Ah, another thing I forgot to revert. I added that to try and figure out what was causing the `null times zero` error.